package.json

```diff
@@ -3464,7 +3465,7 @@
             "aws-stepfunctions-preview": {
                 "description": "AWS Contributed Icon",
                 "default": {
-                    "fontPath": "./resources/fonts/aws-toolkit-icons.woff",
+                    "fontPath": "./resources\\fonts\\aws-toolkit-icons.woff",
```

Comment:
why was this changed? if `\` slashes are being used somewhere, that should be fixed.

Reply:
Oh interesting, was this a manual change? Might need to update `scripts/build/generateIcons.ts` to only use posix paths

Reply:
No, this was not manual. They all were changed and I had to revert them back. I missed this one. Fixed in next commit.